test/Base_Tests/src/Semantic/Multi_Value_As_Type_Refinement_Spec.enso

```diff
@@ -165,7 +165,7 @@ add_specs suite_builder =
             (c:A&B&C).b_method . should_equal "B method"
             (c:A&B&C).c_method . should_equal "C method"
 
-        group_builder.specify "default to_text should delegate to one of values" pending="TODO: https://github.com/enso-org/enso/issues/11827" <|
+        group_builder.specify "default to_text should delegate to one of values" pending="To text doesn't dispatch properly #11827" <|
```

Comment:
If this PR fixes #11827 then why is this test still marked as pending?

Does that mean that #11827 is not resolved and shouldn't be closed? Or should we unmark this test and start running it?